### DISCUSSION TITLE
Unable to see multiple results

### DISCUSSION THREAD

**Discussion by techlock77** [dbeaver/dbeaver#8945] [Q&A] [answered]
Please fill in as much of the template as possible.
-->

#### System information: 
- Operating system (distribution) and version -Win Server 2012
- DBeaver version - 7.1.0

#### Connection specification:
- Database name and version - Netezza

#### Describe your question:
I am running multiple sql statements like,

select 1;
select 2;
select 3;
select 4;

But, in the results tab, I am getting results from the first statement ONLY.

Instead, I want to have results from all the four statements side-by-side (Horizontally). Do I have to change anything in my Window>Preferences tab?

![image](https://user-images.githubusercontent.com/66756251/84329579-099ada00-ab53-11ea-8411-3e18c4f65093.png)

**Comment by uslss** [comment, marked as answer]
Hi @techlock77 ,
you could use Execute SQL Script button
![image](https://user-images.githubusercontent.com/31996417/84401431-ee3fd580-ac0b-11ea-974c-fb2f7d55dc89.png)
